tools/repeat_channels/repeat_channels.xml

```diff
@@ -0,0 +1,61 @@
+<tool id="repeat_channels" name="Convert single-channel to multi-channel image" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="22.05">
+    <description>with NumPy</description>
```

Comment:
Is numpy interesting for the users.

Reply:
Not sure, this tool is a very thin wrapper around `numpy.dstack`, and, regarding what we agreed as a community, this seems to be an edge case:

> If a Galaxy tool is a thin tool wrapper (e.g, part of a major tool suite), then the name of the wrapped tool (and only the name of the wrapped tool, subsequent to the term “with” as in “with Bioformats”) should be used as the description of the tool […] **The tool description should be empty if a tool is either not part of a major tool suite**, or the main functionality of the tool is implemented in the wrapper.

One could argue that NumPy is a tool suite…